pvt/benchmarks/joinExit.ts

```diff
@@ -59,7 +59,7 @@ async function main() {
     );
   }
   console.log('\n');
-  const maxInvestmentTokens = 93;
+  const maxInvestmentTokens = 92;
```

Comment:
What's this about?

Reply:
The aforementioned "breathing on it" apparently increases the cost to the point where 93 tokens fail. As the contracts get bigger, the max tokens you can initialize with decreases. In the latest PR It's down to 81 (maybe 80 now).